### ISSUE TITLE
Incompatibility with mathastext and XeLaTeX

### ISSUE THREAD

**Issue by michaelroland** [michaelroland/jku-templates-presentation-latex#12]
Due to the latest updates to mathastext, the template no longer compiles and fails with

```
! Extended mathchar used as mathchar (4203026).
<to be read again> 
                   }
l.1241 ...letters\string-\mathbin\mst@varfam@minus
                                                  
A mathchar number must be between 0 and "7FFF.
```

See https://github.com/michaelroland/jku-templates-presentation-latex/commit/eea692c8dfa56ceb2d02f6f72c6ff756b3482f25

**Comment by michaelroland** [comment]
This can be overcome by providing the deprecated option `everymath` to the package mathastext.

**Comment by michaelroland** [comment]
Issue has been acknowledged by mathastext author via personal e-mail. Proposal was to stick with everymath for now. A fix for mathastext will be provided before removal of the deprecated everymath option.

For now, this is resolved by #14
